concepts/bindings/README.md

```diff
@@ -1,12 +1,12 @@
 # Bindings
 
-Using bindings, you can trigger your app with events coming in from external systems, or invoke external systems.
-Bindings allow for on-demand, event-driven compute scenarios, and dapr bindings help developers with the following:
+Dapr provides a mechanism that can either (a) trigger your app in response to an event or (b) trigger an external system when your app does something.
 
-* Remove the complexities of connecting to, and polling from, messaging systems such as queues, message buses, etc.
-* Focus on business logic and not the implementation details of how interact with a system
-* Keep the code free from SDKs or libraries
```

Comment:
Any reason that this benefits was remove for the code free of SDKs?

Reply:
@msfussell I moved the mention of SDK up to the second paragraph of the doc. I'll add it back here to reinforce it as well